apps/api/src/routers/user.py

```diff
@@ -150,6 +150,7 @@ async def _apply_flow(
     processed_application_data = ProcessedApplicationDataUnionAdapter.validate_python(
         {
             **raw_app_data_dump,
+            "email": user.email,
```

Comment:
when merged with main, this should be added in both hackers/mentors and volunteer handlers